LICENSE

```diff
@@ -1,8 +1,4 @@
-# LICENSE
-
-zlib License
-
-Copyright (C) 2020 Animal Crossing Music Extension
+Copyright (C) 2020 PikaDude
```

Comment:
I didn't know what to put here so I just put @PikaDude 's username as this is living in his repo 🤷‍♂️

Reply:
I'm not sure if it would be valid to do this.

Reply:
Actually I did some research at the beginning of the fork and it looks like best way to procede is to put current copyright holders first then include the original line as 'partial copyright'

Reply:
I'll discard this change seeing how we will probably publish it under the ACME gh org (wow I like that acronym!)